React/todoApp/src/app/store.js

```diff
@@ -0,0 +1,35 @@
+import { configureStore } from "@reduxjs/toolkit";
+import todosReducer from "../features/todos/todosSlice";
+
+function loadFromStorage() {
+  try {
+    const raw = localStorage.getItem("todos_state");
```

Comment:
maybe we can give a more meaningful name.

```suggestion
    const todos = localStorage.getItem("todos_state");
```